docs/guides/theme_architecture/page_layouts/index.md

```diff
@@ -3,3 +3,43 @@ layout: default
 title: Page Layouts
 parent: Theme Architecture
 ---
+
+Page Layouts are predefined collections of blocks to help Creators quickly get going with a sensible layout. Each layout is defined in its own json file in the /pages folder of your theme and should have:
+- a name
+- a category 
+- a collection of blocks 
+
+## Name 
+The page name is shown when choosing a page and also determines the name and link of the page that is created. If the link are already in use they will be suffixed with the next available number, however sites can have pages with the same name.
```

Comment:
Usage of singular and plural seems inconsistent (_If the link_)?

Reply:
Nice spot thanks

Reply:
How about _'The page name is shown when choosing a page and also determines the name and slug of the page that is created. As slugs must be unique, if the slug is already in use it will be suffixed with the next available number, however sites can have pages with the same name.'_